public/app/features/alerting/unified/components/rule-editor/alert-rule-form/ModifyExportRuleForm.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { Link } from 'react-router-dom';
+
+import { Stack } from '@grafana/experimental';
+import { Button, CustomScrollbar } from '@grafana/ui';
+
+import { AppChromeUpdate } from '../../../../../../core/components/AppChrome/AppChromeUpdate';
+import { RuleFormValues } from '../../../types/rule-form';
+import { MINUTE } from '../../../utils/rule-form';
+import { GrafanaExportDrawer } from '../../export/GrafanaExportDrawer';
+import { allGrafanaExportProviders, ExportFormats } from '../../export/providers';
+import { AlertRuleNameInput } from '../AlertRuleNameInput';
+import AnnotationsStep from '../AnnotationsStep';
+import { GrafanaEvaluationBehavior } from '../GrafanaEvaluationBehavior';
+import { NotificationsStep } from '../NotificationsStep';
+import { QueryAndExpressionsStep } from '../query-and-alert-condition/QueryAndExpressionsStep';
+
+interface ModifyExportRuleFormProps {
+  alertUid?: string;
+  ruleForm?: RuleFormValues;
+}
+
+type ModifyExportMode = 'rule' | 'group';
+
+export function ModifyExportRuleForm({ ruleForm, alertUid }: ModifyExportRuleFormProps) {
+  const formAPI = useForm<RuleFormValues>({
+    mode: 'onSubmit',
+    defaultValues: ruleForm,
+    shouldFocusError: true,
+  });
+
+  const existing = Boolean(ruleForm);
+  const returnTo = `/alerting/list`;
+
+  const [showExporter, setShowExporter] = useState<ModifyExportMode | undefined>(undefined);
+
+  const [conditionErrorMsg, setConditionErrorMsg] = useState('');
+  console.log('conditionErrorMsg', conditionErrorMsg);
+  const [evaluateEvery, setEvaluateEvery] = useState(ruleForm?.evaluateEvery ?? MINUTE);
+
+  const checkAlertCondition = (msg = '') => {
+    setConditionErrorMsg(msg);
+  };
+
+  const actionButtons = [
+    <Link to={returnTo} key="cancel">
+      <Button size="sm" variant="secondary" onClick={() => null}>
```

Comment:
Doesn't the `LinkButton` work here so that we can avoid the `onClick={() => null}` part?

Reply:
True!, i will use LinkButton component instead 👍